sass/srobo/variables/colors.scss

```diff
@@ -1,5 +1,5 @@
-$sr-blue:        rgba(36, 51, 115, 1);
```

Comment:
As much as I personally prefer hex, I think we should keep these as `rgba` to match the rest